opal/mca/btl/base/btl_base_am_rdma.c

```diff
@@ -1138,5 +1138,28 @@ int mca_btl_base_am_rdma_init(mca_btl_base_module_t *btl)
         OBJ_CONSTRUCT(&default_module, mca_btl_base_am_rdma_module_t);
     }
 
+    /* This section check whether we can claim support of remote completion.
```

Comment:
I don't think you need this first line of the comment.  But I do love the detailed comment!